pkg/webhook/conversion/conversion.go

```diff
@@ -69,16 +70,21 @@ func (wh *Webhook) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
```

Comment:
I would prefer handling the convertReview.Request == nil explicitly and return StatusBadRequest in this case like in ~ l. 68-70) (like you suggested as an alternative)